src/Wordpress/PluginActions.php

```diff
@@ -52,4 +52,65 @@ public function loginWordPress()
 
         return $response;
     }
+
+    /**
+     * GET /plugin/:zonedId/settings.
+     *
+     * @return mixed
+     */
+    public function getPluginSettings()
+    {
+        // Always return true
+        $value = true;//$this->dataStore->getIpRewrite();
```

Comment:
Why does this always return `true`?  Shouldn't we always be returning `$this->dataStore->getIpRewrite();`